backend/app/views/backend/users/_form.html.slim

```diff
@@ -22,7 +22,19 @@ div.fields
 
   = render 'backend/shared/form/field_textarea', form: form, value: :description
 
-  = render 'backend/shared/form/field_input', form: form, value: :web_url
+  = render 'backend/shared/form/field_input', form: form, value: :web_url, label_txt: 'Web'
+
+  = render 'backend/shared/form/field_input', form: form, value: :facebook_url, label_txt: 'Facebook'
+
+  = render 'backend/shared/form/field_input', form: form, value: :linkedin_url, label_txt: 'LinkedIn'
+
+  = render 'backend/shared/form/field_input', form: form, value: :skype_user, label_txt: 'Skype User'
+
+  = render 'backend/shared/form/field_input', form: form, value: :twitter_url, label_txt: 'Twitter'
+
+  = render 'backend/shared/form/field_input', form: form, value: :google_url, label_txt: 'Google+'
+
+  = render 'backend/shared/form/field_input', form: form, value: :video_id, label_txt: 'Video ID', note_txt: "Set the Vimeo video id"
 
   = render 'backend/shared/form/field_select', form: form, value: :role, collection: user_roles, as: :collection_select, label_txt: 'Role'
```

Comment:
Let's swap the position of roles with the social bits, as I feel it's more relevant for it to be at top, now that there are so many links.